packages/pigment-css-react/src/base.ts

```diff
@@ -64,3 +65,10 @@ export interface PolymorphicComponent<SxProp, BaseProps extends BaseDefaultProps
     props: PolymorphicComponentProps<SxProp, BaseProps, AsTarget>,
   ): JSX.Element;
 }
+
+export interface BreakpointOverrides {}
+
+export type Breakpoint = OverridableStringUnion<
+  'xs' | 'sm' | 'md' | 'lg' | 'xl',
```

Comment:
Seeing this makes me think of https://github.com/mui/material-ui/issues/26168

Reply:
We'll still need to continue supporting this unless we want breaking changes.
We could also introduce a way to alias breakpoints, ie, `xs -> mobile` etc.